downloader.py

```diff
@@ -354,21 +363,20 @@ def main(argv):
             # get the list of videos
             courses_page = session.get("https://www.packtpub.com/account/my-courses", verify=True, headers=headers)
             courses_tree = html.fromstring(courses_page.content)
-            course_nodes = courses_tree.xpath("//div[@id='product-account-list']/div[contains(@class,'product-line unseen')]")
+            course_nodes = courses_tree.xpath(
+                "//div[@id='product-account-list']/div[contains(@class,'product-line unseen')]")
 
             print('###########################################################################')
             print("FOUND {0} INTEGRATED COURSES: STARTING DOWNLOADS".format(len(course_nodes)))
             print('###########################################################################')
 
             # loop through the videos
             for course in course_nodes:
-
+                # TODO: enumerate(course_nodes) or add counter to print progress indicator (e.g. downloading course 5/24)
                 # download the book
                 courses_directory = os.path.join(root_directory, "courses")
                 download_course(course, courses_directory, course_assets, session, headers)
 
 
 if __name__ == "__main__":
-    reload(sys)
```

Comment:
This caused an error in the global and the virtual environment using Python 3.6.
This seems to be deprecated and discouraged now... was probably used with iPython notebook.